include/bcs/MFEMScalarFunctorBoundaryIntegratedBC.h

```diff
@@ -1,12 +1,12 @@
 #pragma once
```

Comment:
Why did you rename a vector BC to a scalar BC? Wouldn't it have made more sense to keep vectors and scalars the same?